### DISCUSSION TITLE
Dementonite from mindsunraveled event can not be used to fabricate items

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15187] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

it has nameidentifier="dementonite"

### Reproduction steps

1. try to use event exclusive dementonite to fabricate items

### Bug prevalence

Just once

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
instead of creating a confusing variant use StatusEffectAction to implement sound and visual effect?

**Comment by JeroonBosch** [comment]
Thank you for your report, in the future this item cannot be picked up anymore.
